### PR TITLE
LPD-23925 Create new configuration method on "WorkflowDefinitionConfiguration"

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPD-23925](https://liferay.atlassian.net/browse/LPD-23925)

### CODE REVIEW

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hey @diogomelo09, see my comment,

Nice job!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ allow=Allow
 allow-adding-pages=Allow Adding Pages
 allow-administrator-to-create-and-execute-code-in-liferay=Allow administrator to create and execute code in Liferay.
 allow-administrators-to-publish-and-edit-workflows=Allow Administrators to Publish and Edit Workflows
-allow-administrators-to-publish-and-edit-workflows-description=When selected, users with the Regular Administrator role in a virtual instance can publish new workflow definitions, including scripting content that is executed inside the portal (and potentially in other virtual instances).
+allow-administrators-to-publish-and-edit-workflows-description=When selected, users with the Regular Administrator role in a virtual instance can publish new workflow definitions.
```
[the remark this answers is not in the document]
Comment (diogomelo09) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: Done :white_check_mark:

File: modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/configuration/WorkflowDefinitionConfiguration.java [hunk 2]
```diff
@@ -29,4 +29,12 @@ public interface WorkflowDefinitionConfiguration {
 	)
 	public boolean companyAdministratorCanPublish();
 
+	@Meta.AD(
+		deflt = "false",
+		description = "notifications-must-honor-site-membership-rule-description",
+		id = "disable.notifications.parent.site.members",
+		name = "notifications-must-honor-site-membership-rule", required = false
+	)
+	public boolean disableNotificationsParentSiteMember();
```
[the remark this answers is not in the document]
Comment (diogomelo09) [reply modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/configuration/WorkflowDefinitionConfiguration.java]: Done :white_check_mark:

[the remark this answers is not in the document]
Comment (diogomelo09) [reply modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/configuration/WorkflowDefinitionConfiguration.java]: I withdrew the ID, as agreed in a conversation with @MarinhoFeliphe

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by diogomelo09** [comment]
ci:test:relevant

**Comment by diogomelo09** [comment]
ci:forward

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by diogomelo09** [comment]
ci:test:stop

**Comment by diogomelo09** [comment]
ci:forward

**Comment by diogomelo09** [comment]
ci:test:sf

**Comment by diogomelo09** [comment]
ci:stop

**Comment by diogomelo09** [comment]
ci:forward

**Comment by diogomelo09** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/149628

**Pull request closed without merging**